zipline/algorithm.py

```diff
@@ -1461,8 +1888,13 @@ def validate_account_controls(self):
 
     @api_method
     def set_max_leverage(self, max_leverage=None):
-        """
-        Set a limit on the maximum leverage of the algorithm.
+        """Set a limit on the maximum leverage of the algorithm.
+
+        Parameters
+        ----------
+        max_leverage : float, optional
+            The maximum leverage for the algorithm. If not provided there will
+            be no maximum.
```

Comment:
I agree, I don't think that we should have as many defaults as we do but I didn't want to change the semantics with this PR. I was only trying to document what exists.